docker-compose.yml

```diff
@@ -1,9 +1,9 @@
-# Run tests in Docker `docker-compose up` on Node.js v4
+# Run tests in Docker `docker-compose up` on Node.js v14
```

Comment:
Might be safe to just drop the version in comment since it's hard to keep updated. :-)